test/unittests.jl

```diff
@@ -395,17 +404,19 @@ end
     end
 
     @testset "Multiplying ranges with units" begin
+        T_QA_StepRangeLen = QuantityArray{T, 1, D, Q, V} where {T, D<:Dimensions, Q<:UnionAbstractQuantity, V<:StepRangeLen}
+        T_QA_s_StepRangeLen = QuantityArray{T, 1, D, Q, V} where {T, D<:SymbolicDimensions, Q<:UnionAbstractQuantity, V<:StepRangeLen}
         # Test multiplying ranges with units
         x = (1:0.25:4)u"inch"
-        @test x isa StepRangeLen
+        @test x isa T_QA_StepRangeLen
```

Comment:
```suggestion
        @test x isa QuantityArray
        @test DQ.array_type(x) isa StepRangeLen
        @test DQ.dim_type(x) isa Dimensions
```

bit more readable like this.

Same comment for other `T_QA_AbstractArray` types - we can swap them for using the interface to inspect the type parameters

Reply:
Ok, will swap these out. Sorry, by using the interface to inspect type parameters, is there something else that you would like to see instead?

Reply:
017d5d4